tests/e2e/k8s/snapshotter-cri.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: nydus-system
+---
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nydus-snapshotter-sa
+  namespace: nydus-system
+
+---
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nydus-snapshotter
+  namespace: nydus-system
+spec:
+  containers:
+  - env:
+    - name: ENABLE_NYDUS_OVERLAY
+      value: "false"
+    name: nydus-snapshotter
+    image: local-dev:e2e
+    imagePullPolicy: IfNotPresent
+    args:
+      - containerd-nydus-grpc
+      - --nydusd-path /usr/local/bin/nydusd
+      - --config-path /etc/nydus/config.json
+      - --root /var/lib/containerd-nydus
+      - --address /run/containerd-nydus/containerd-nydus-grpc.sock
+      - --log-level debug
+      - --daemon-mode shared
+      - --log-to-stdout
+      - --enable-cri-keychain
```

Comment:
Is this `--enable-cri-keychain` needed? this flag will enable Kubernetes secrets listener to watch secrects. For `cri` type, is it needed?